pandas/core/ops.py

```diff
@@ -1591,4 +1583,5 @@ def wrapper(self, other):
 
 sparse_series_special_funcs = dict(arith_method=_arith_method_SPARSE_SERIES,
                                    comp_method=_arith_method_SPARSE_SERIES,
-                                   bool_method=None)
+                                   bool_method=_bool_method_SERIES)
```

Comment:
can we hold off adding till then

Reply:
This is necessary to get rid of the double-call in sparse.series.